lib/membrane_http_adaptive_stream/sink_bin.ex

```diff
@@ -202,26 +201,39 @@ defmodule Membrane.HTTPAdaptiveStream.SinkBin do
           ]
 
         state.mode == :muxed_av and encoding == :H264 ->
+          audio_tee_links =
```

Comment:
that won't work because cmaf in `:muxed_av` mode has to get all inputs pads in the same action. Cmaf has to create a header when it gets all stream formats.